sentry_sdk/integrations/quart.py

```diff
@@ -27,11 +27,12 @@
 
 try:
     from quart import (  # type: ignore
+        has_request_context,
```

Comment:
This does not work with Quart 0.16.1 (the first version we support). We need to be backwards compatible

Could you change the imports to something like this: 
https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/integrations/starlette.py#L43-L47

Where you then down on line 112 can check do something like:
```
if has_request_context is None:
    # old way of getting request
else:
    # new way of getting request
```

Reply:
Are you sure it doesn't work with 0.16.1, `has_request_context` is present in that release, https://github.com/pallets/quart/blob/0.16.1/src/quart/__init__.py#L15

Reply:
But it can not be imported from quart as in master, because the "as xxx" is missing: https://github.com/pallets/quart/blob/main/src/quart/__init__.py#L15

Reply:
Sorry, I'm not following. In [0.16.1 an `__all__`](https://github.com/pallets/quart/blob/0.16.1/src/quart/__init__.py#L87) is used rather than `as xxx`. It is only the latest, 0.18.0 that uses the `as xxx` syntax.

I've just installed 0.16.1 and can import it.

Reply:
Hey @pgjones !
I have now checked it again, and you are right. It works...
I don't know why it did not work when I was testing it the first time. 
So I guess this is fine!